README.md

```diff
@@ -1,6 +1,12 @@
 # Listable
 
-This Neos package solves one problem: help you list any nodes in TypoScript.
+For the neos user interface, this package provides a content element "List" (experimental) that can render a list of nodes with a ListableMixin that can be added to node types that you want to list.
```

Comment:
List is an experimental things, I'm not sure it should be included at all and surely it should not be listed first when describing this package, it's just not what it is about.